tasks/main.yml

```diff
@@ -24,6 +24,48 @@
   when:
     - network_state is defined
     - ansible_distribution_major_version | int < 8
+
+- name: Check if updates for network packages are available through the DNF
+    package manager due to wireless or team interfaces
+  dnf:
+    update_cache: true
+    name: "{{ network_packages }}"
+    state: latest  # noqa package-latest
```

Comment:
There needs to be an explanation why this needs to be latest. Ideally in the commit message IMHO.

Reply:
I explained in the commit message.